src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailServiceTest.java

```diff
@@ -96,6 +88,96 @@ void sendObjectionSubmittedCustomerEmail() throws ServiceException {
         assertTrue(data.containsValue("Company: " + COMPANY_NUMBER));
         assertTrue(data.containsValue(OBJECTION_ID));
         assertTrue(data.containsValue(attachments));
+    }
+
+    @Test
+    void sendObjectionSubmittedDissolutionEmailsWalesJurisdiction() throws ServiceException {
+        when(dateTimeSupplier.get()).thenReturn(LOCAL_DATE_TIME);
+        when(config.getEmailRecipientsCardiff()).thenReturn(EMAIL_RECIPIENTS_CARDIFF_TEST);
+        Objection objection = Utils.getTestObjection(OBJECTION_ID);
+        Utils.getTestAttachments().forEach(objection::addAttachment);
+
+        emailService.sendObjectionSubmittedDissolutionTeamEmail(
+                REQUEST_ID,
+                Utils.getDummyCompanyProfile(COMPANY_NUMBER, "wales"),
+                objection
+        );
+
+        ArgumentCaptor<EmailContent> emailContentArgumentCaptor = ArgumentCaptor.forClass(EmailContent.class);
+        verify(kafkaEmailClient, times(3)).sendEmailToKafka(emailContentArgumentCaptor.capture());
+    }
+
+    @Test
+    void sendObjectionSubmittedDissolutionEmailsScotlandJurisdiction() throws ServiceException {
+        when(dateTimeSupplier.get()).thenReturn(LOCAL_DATE_TIME);
+        when(config.getEmailRecipientsEdinburgh()).thenReturn(EMAIL_RECIPIENTS_EDINBURGH_TEST);
+        Objection objection = Utils.getTestObjection(OBJECTION_ID);
+        Utils.getTestAttachments().forEach(objection::addAttachment);
 
+        emailService.sendObjectionSubmittedDissolutionTeamEmail(
+                REQUEST_ID,
+                Utils.getDummyCompanyProfile(COMPANY_NUMBER, "scotland"),
+                objection
+        );
+
+        ArgumentCaptor<EmailContent> emailContentArgumentCaptor = ArgumentCaptor.forClass(EmailContent.class);
+        verify(kafkaEmailClient, times(2)).sendEmailToKafka(emailContentArgumentCaptor.capture());
+    }
+
+    @Test
+    void sendObjectionSubmittedDissolutionEmailsNIJurisdiction() throws ServiceException {
+        when(dateTimeSupplier.get()).thenReturn(LOCAL_DATE_TIME);
+        when(config.getEmailRecipientsBelfast()).thenReturn(EMAIL_RECIPIENTS_BELFAST_TEST);
+        Objection objection = Utils.getTestObjection(OBJECTION_ID);
+        Utils.getTestAttachments().forEach(objection::addAttachment);
+
+        emailService.sendObjectionSubmittedDissolutionTeamEmail(
+                REQUEST_ID,
+                Utils.getDummyCompanyProfile(COMPANY_NUMBER, "northern-ireland"),
+                objection
+        );
+
+        ArgumentCaptor<EmailContent> emailContentArgumentCaptor = ArgumentCaptor.forClass(EmailContent.class);
+        verify(kafkaEmailClient, times(2)).sendEmailToKafka(emailContentArgumentCaptor.capture());
+    }
+
+    @Test
+    public void testRegionalEmailAddresses() {
```

Comment:
you can remove 'public' to keep it consistent with the other tests in this class

Reply:
Removed.